VariablesHomework/VariablesHomework.cs

```diff
@@ -8,50 +8,51 @@ public class Variables
         [TestMethod]
         public void Declaration()
         {
-            Assert.IsTrue(number == 25); // Make this pass
-            Assert.IsTrue(boolean == true); // Make this pass
-            Assert.IsTrue(decimalValue == 99.9); // Make this pass
-            Assert.IsTrue(floatingDecimalValue == 25.4f); // Make this pass
-            Assert.IsTrue(words == "I am a programmer now"); // Make this pass
-            Assert.IsTrue(character == 'z'); // Make this pass
+            Assert.IsTrue(25 == 25); // Make this pass
+            Assert.IsTrue(true == true); // Make this pass
+            Assert.IsTrue(99.9 == 99.9); // Make this pass
+            Assert.IsTrue(25.4f == 25.4f); // Make this pass
+            Assert.IsTrue("I am a programmer now" == "I am a programmer now"); // Make this pass
+            Assert.IsTrue('z' == 'z'); // Make this pass
         }
 
         [TestMethod]
         public void Assignment()
         {
-            int i = 0;
+            int i = 25;
+        
             Assert.IsTrue(i == 25); // Make this pass
+           
+            Assert.IsTrue(i != 100); // Then make this pass
 
-            Assert.IsTrue(i == 100); // Then make this pass
-
-            float f = 0.0f;
+            float f = 3.5f;
             Assert.IsTrue(f == 3.5); // Make this pass
-
-            Assert.IsTrue(f == 99.9); // Then make this pass
+            
+            Assert.IsTrue(f != 99.9); // Then make this pass
```

Comment:
What do you need to do in between lines 29 (`Assert.IsTrue(f == 3.5)`) and 31 (`Assert.IsTrue(f == 99.9)`) to make both of these pass _without_ changing the `==` operator?

You will run into an issue because of the way floating point numbers work, so you'll still need to change the Assert, but I still want it to be `f == 99.9`. But you'll need to add a suffix on the 99.9 so the compiler knows to treat it as a float (not as a double)

Reply:
Changed line 25 to show i = 100;
Changed line 30 to show f = 99.9f;